functions.php

```diff
@@ -306,3 +306,61 @@ function get_first_post_image() {
  * Remove native Gallery styling
  */
 add_filter( 'use_default_gallery_style', '__return_false' );
+
+
+
+/**
+ * Address CPTonomies plugin bug
+ *
+ * @param type $post post.
+ * @param type $taxonomy taxonomy.
+ */
+function get_the_terms_override( $post, $taxonomy ) {
```

Comment:
The function comment for this indicates that it fixes a bug that IIRC is specific to the MOH theme? Or am I mis-remembering? If this _is_ about an MOH problem, I think it can probably be removed from this PR. No sites on the child theme currently use the CPTonomies plugin.

This will also remove the only thing being flagged by Travis, and the only bug that I care about that is coming from CodeClimate.

Reply:
Yes thanks for catching this. it is exclusively for the moh theme , I will update the file